Note container runtime requirement for image volume subPath (#56606)
kubernetes/website: content/en/docs/concepts/storage/volumes.md

- [`subPath`](/docs/concepts/storage/volumes/#using-subpath) or
  [`subPathExpr`](/docs/concepts/storage/volumes/#using-subpath-expanded-environment)
  mounts for containers (`spec.containers[*].volumeMounts[*].subPath`, `spec.containers[*].volumeMounts[*].subPathExpr`)

>   are only supported from Kubernetes v1.33, and are applied by the container

Reviewer: nit: passive voice, though it continues the voice of the existing bullet.

```suggestion
  are only supported from Kubernetes v1.33. The container runtime applies them:
```
